namerd/docs/interface.md

```diff
@@ -41,8 +41,10 @@ Key | Default Value | Description
 -------------- | -------------- | --------------
 bindingCacheActive | `1000` | The size of the binding active cache.
 bindingCacheInactive | `100` | The size of the binding inactive cache.
+bindingCacheInactiveTTLSecs | `604800` | The amount of time, in seconds, to keep objects in the binding inactive cache before expiring them.
```

Comment:
should this indicate that the default is 600 now?

Reply:
You're right.